docs/helpers/html.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Renders as HTML inside a React `div`.
+
+ * @param name Tag name, to use as root element.
+ * @param param1 Properties and attributes.
```

Comment:
What's `param1`?

Reply:
The unnamed second argument, since I'm destructuring it inline.
It goes `function (param0, param1, param2, ...) {}`

The only other way to document that argument is to name it, then destructure in the function's body.